odoo_updates/odoo_updates.py

```diff
@@ -266,6 +350,53 @@ def diff_to_screen(views_states, title):
                     click.secho(line)
 
 
+def fields_to_screen(fields_states, title):
+    show_model_field = dict()
+    for state, values in fields_states.iteritems():
+        click.secho('+ {state} {title}'.
+                    format(state=state.title(), title=title), fg='yellow')
+        for field in values:
+            if state == 'updated':
+                column = {'type': difflib.unified_diff(
+                        field['original'].get('type', '').split('\n'),
+                        field['modified'].get('type', '').split('\n')),
+                        'field_description': difflib.unified_diff(
+                            field['original'].get('field_description', '').split('\n'),
+                            field['modified'].get('field_description', '').split('\n')), }
+                field.update(column)
```

Comment:
@mapuerta why do you add the diff to the field dictionary? you just need the diff to display it in the screen, no need to add it to the field dictionary